aya/src/programs/fexit.rs

```diff
@@ -0,0 +1,79 @@
+//! fexit programs.
+use thiserror::Error;
+
+use crate::{
+    generated::{bpf_attach_type::BPF_TRACE_FEXIT, bpf_prog_type::BPF_PROG_TYPE_TRACING},
+    obj::btf::{Btf, BtfError, BtfKind},
+    programs::{load_program, utils::attach_btf_id, LinkRef, ProgramData, ProgramError},
+};
+
+/// Marks a function as a fexit eBPF program that can be attached to almost
```

Comment:
Same as fentry, please change the doc format.

Reply:
done